.ci/deps.sh

```diff
@@ -99,3 +99,6 @@ fi
 # Tailor (Swift) commands
 curl -fsSL https://tailor.sh/install.sh | sed 's/read -r CONTINUE < \/dev\/tty/CONTINUE=y/' > install.sh
 sudo bash install.sh
+
+# FlawFinder will be confused by our python 3 venv
+sudo sed -i 's/env python/python2/' /usr/bin/flawfinder
```

Comment:
Is this ok ?
I honestly don't like things like this ... It's meddling with things it sholdnt.

I'd prefer if we make a `.ci/install.flawfinder.sh` which simply installs it using [this](https://gist.github.com/FlorianWolters/07c866500a1fe6f6098c#flawfinder).

@sils1297 ? Is the shebang hack ok with you ?